src/monodroid/monodroid.targets

```diff
@@ -22,22 +19,33 @@
         Overwrite="True"
     />
     <Exec
-        Command="$(AndroidToolchainDirectory)\ndk\ndk-build CONFIGURATION=$(Configuration) V=1"
+        Command="$(AndroidToolchainDirectory)\ndk\ndk-build CONFIGURATION=Debug V=1"
     />
     <Copy
         SourceFiles="@(AndroidSupportedTargetJitAbi->'obj\local\%(Identity)\libmonodroid.so')"
-        DestinationFiles="@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android.$(_Conf).d.so')"
+        DestinationFiles="@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android.debug.d.so')"
     />
     <Copy
         SourceFiles="@(AndroidSupportedTargetJitAbi->'libs\%(Identity)\libmonodroid.so')"
-        DestinationFiles="@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android.$(_Conf).so')"
+        DestinationFiles="@(AndroidSupportedTargetJitAbi->'$(OutputPath)\%(Identity)\libmono-android.debug.so')"
+    />
+    <Exec
+        Command="$(AndroidToolchainDirectory)\ndk\ndk-build CONFIGURATION=Release V=1"
```

Comment:
Do we need to do a "clean" or something before re-running `ndk-build`, to ensure that everything is rebuilt? I fear/suspect that simply changing an environment variable won't cause everything to be recompiled.

The "best" fix would be for Debug and Release object files to go in separate directories, but I couldn't easily determine how to control that within `ndk-build`... :-/